t/043-shdict.t

```diff
@@ -2298,3 +2298,100 @@ foo = nil
 --- no_error_log
 [error]
 
+
+
+=== TEST 86: the lightuserdata ngx.null hasn't methods of shared dicts.
```

Comment:
This test does not exercise the bug and also passes successfully with the current master. This is because you did not define any `lua_shared_dict` in this test case, thus no metatables registered on any light user data.

This is why we need to run the tests again the old code version to ensure that they actually fail :) Tests that always pass are meaningless :)